pkg/controllers/curvecluster_controller.go

```diff
@@ -138,9 +138,7 @@ func (r *CurveClusterReconciler) reconcileDelete(curveCluster *curvev1.CurveClus
 	}
 
 	// Delete it from clusterMap
-	if _, ok := r.ClusterController.clusterMap[curveCluster.Namespace]; ok {
-		delete(r.ClusterController.clusterMap, curveCluster.Namespace)
-	}
+	delete(r.ClusterController.clusterMap, curveCluster.Namespace)
```

Comment:
You're right, but it's more more intuitive using if.

Reply:
yes, but it may be more obvious to developers and me.

Reply:
> yes, but it may be more obvious to developers and me.

Okay, I'll revert it.

Reply:
Done. PTAL